intl/msg_hash_us.h

```diff
@@ -1059,21 +1059,21 @@ MSG_HASH(MENU_ENUM_LABEL_VALUE_RDB_ENTRY_TGDB_RATING,
 MSG_HASH(MENU_ENUM_LABEL_VALUE_REBOOT,
       "Reboot")
 MSG_HASH(MENU_ENUM_LABEL_VALUE_RECORDING_CONFIG_DIRECTORY,
-      "Recording Config Dir")
+      "Recording Config (dir)")
 MSG_HASH(MENU_ENUM_LABEL_VALUE_RECORDING_OUTPUT_DIRECTORY,
-      "Recording Output Dir")
+      "Recording Output (dir)")
 MSG_HASH(MENU_ENUM_LABEL_VALUE_RECORDING_SETTINGS,
       "Recording")
 MSG_HASH(MENU_ENUM_LABEL_VALUE_RECORD_CONFIG,
-      "Record Config")
+      "Load Record Config")
 MSG_HASH(MENU_ENUM_LABEL_VALUE_RECORD_DRIVER,
       "Record Driver")
 MSG_HASH(MENU_ENUM_LABEL_VALUE_RECORD_ENABLE,
-      "Record Enable")
+      "Enable Recording")
 MSG_HASH(MENU_ENUM_LABEL_VALUE_RECORD_PATH,
-      "Output File")
+      "Save Record as...")
```

Comment:
it's not Save Record as.. it's output file, it doesn't save a t the moment you select the option.